lib/worker/app.js

```diff
@@ -2,21 +2,19 @@
 //        instance, mainly to use the `auth()` method. Probot needs refactored
 //        so that method is more easily accessible.
 
-const { createProbot } = require('probot');
-const { findPrivateKey } = require('probot/lib/private-key');
+const { Server, Probot } = require('probot');
+const { getPrivateKey } = require('@probot/get-private-key');
 
-const probot = createProbot({
-  id: process.env.APP_ID,
-  cert: findPrivateKey(),
-
-  // These aren't needed by worker process
-  secret: undefined,
-  port: undefined,
-  webhookPath: undefined,
-  webhookProxy: undefined,
+const server = new Server({
+  Probot: Probot.defaults({
+    appId: process.env.APP_ID,
+    privateKey: getPrivateKey(),
+    logLevel: process.env.LOG_LEVEL,
+  }),
 });
 
 // Load an empty app so we can get access to probot's auth handling
-const app = probot.load(() => {});
+server.load(() => {});
+const app = server.probotApp;
 
 module.exports = app;
```

Comment:
It seems like you can replace the whole file with

```suggestion
module.exports = new Probot.defaults({
  appId: process.env.APP_ID,
  privateKey: getPrivateKey(),
  logLevel: process.env.LOG_LEVEL,
})
```

Reply:
@gr2m I am getting error for this

```
module.exports = new Probot.defaults({
worker |                  ^
worker | 
worker | TypeError: Probot.defaults is not a constructor
```

Reply:
can you try removing the `.defaults`?

```suggestion
module.exports = new Probot({
  appId: process.env.APP_ID,
  privateKey: getPrivateKey(),
  logLevel: process.env.LOG_LEVEL,
})
```

Reply:
yes this worked. Updated file